### ISSUE TITLE
more detailed exit statuses for `findmnt`

### ISSUE THREAD

**Issue by calestyo** [util-linux/util-linux#2738]
Hey.

Right now `findmount` seems to always use either `0` when it prints something or `1` on any errors or when it just has nothing to print.

IMO it would be useful if there were more detailed exit statuses:
E.g. if I do:
```
findmnt --noheadings --raw --output ID --kernel --source /dev/sr0
```
in order to see whether a CD drive is mounted somwhere and get `1`, it might be at least because of:
1. some internal error (maybe it uses `/proc` an that’s missing or maybe it cannot check the mountpoints because of permissions)
2. `/dev/sr0` not even existing
3. `/dev/sr0` simply not being mounted

To the least it would be handy if one could tell apart (1) from (2) and (3).<br>
But even better, if I could also tell apart (2) and (3) (though I could do that of course with a simple additional check).

So long story short, it would be nice if at least one could tell apart whether there was a real error, or just nothing found.

Thanks,
Chris.
